xml/System.Web/SameSiteMode.xml

```diff
@@ -14,7 +14,20 @@
   </Base>
   <Docs>
     <summary>Specifies constants that indicate the value for the SameSite attribute of the cookie.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks
+
+The <see cref="F:System.Web.SameSiteMode.None" /> value now causes ASP.NET to emit a SameSite cookie header in compliance with https://tools.ietf.org/html/draft-west-cookie-incrementalism-00 that browsers are in the process of adopting. As part of this change, FormsAuth and SessionState cookies will be issued with SameSite = 'Lax' instead of the previous default of 'None', though these values can be overridden in web.config. 
```

Comment:
Does it make sense to mention at this point that in addition to the updated defaults for FormsAuth and SessionState, that all other System.Web.HttpCookies will default to an enum value of -1? Which is not an official member of this enum, but can be specified in config via the string "Unspecified."

Reply:
And would we also want to update the docs for HttpCookie.SameSite as well. (I don't think the current description of the property was correct, even before this update.)
https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Web/HttpCookie.xml#L407

Reply:
Yes, that's a good suggestion. 

Related: would the get accessor on the `HttpCookie.SameSite` property return `-1` in those cases?

Reply:
Regarding `HttpCookie.SameSite`, what else on the description is incorrect? (This is not my area of focus)

Reply:
Yes. The enum is an Int32 enum, so -1 is a valid value. In code, you would cast your enum to (int) and compare to -1 to determine if it is "Unspecified." Comparing to None|Lax|Strict without a cast will still work, even if the enums value is -1.

We couldn't add the full-fledged enum value for "Unspecified" because it updates the API surface. So it's being faked with the -1 value and config interpretations of the string "Unspecified."

Reply:
> Regarding `HttpCookie.SameSite`, what else on the description is incorrect? (This is not my area of focus)

The description says that the default value of that property is 'Lax' if targeting 4.7.2 or later and 'None' otherwise. But the targeting aspect of the original feature got pulled. So it actually doesn't matter what you target, the default value of HttpCookie.SameSite (if it exists at all, since it was only introduced in 4.7.2) was always 'None' regardless of targetFramework.

With this update, that default (for 4.7.2 and 4.8) will change to 'Unspecified' which maintains the same behavior as before when seen on the wire. When seen through server-side code though, the property value will be -1 instead of 'None'.